src/SelfService/Domain/Models/UserStatusCheckerStatus.cs

```diff
@@ -0,0 +1,52 @@
+namespace SelfService.Domain.Models;
+
+public class UserStatusCheckerStatus : ValueObject
```

Comment:
I believe that we are abusing the concept of a ValueObject -- not just here but elsewhere too.
To a certain degree, maybe this abuse was started by Thomas and Jens, but I am not certain.

I don't wan to change this now, but I don't believe this should be a ValueObject, as I don't see the UserStatus being specific to our domain per se (It is something we add outside our domain).

Following this line of thoughts lead me to the belief that DDD is not easily applicable to our specific area of responsibility, but that is another chat entirely.

Reply:
If you look at the values for UserStatusCheckerStatus, you can see that it is somewhat specific to our domain rather than being simple UserStatus (e.g. values like BadAuthToken).

I believe it is pretty in line with what we do elsewhere in the codebase :)